### PR TITLE
fix taskcard ui

### PR DESCRIPTION
**Pull request by shreyachowdhury4**
taskcard2
### Functionality:
fixing task card according to the requirements respectively

### Solution:
The arrangement of the taskcards in a neat manner,colours to the icons acording to the status and tooltips for better understanding.

### Risk level:
- [ ] high 
- [ ]  medium
- [x] low

### How to test:
worktrolly task

### CODE REVIEW

File: worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.html [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
+            <div class="col-md-1 col-1 px-2">
+                <span style="font-weight: bold;" data-toggle="tooltip" data-placement="top" title="task.id">{{task.id}}</span>
             </div>
             <div class="col-md-7 col-4 px-2">
-                <span class="material-icons">Title</span> {{task.Title}}
+                <span class="material-icons" data-toggle="tooltip" data-placement="top" title="Title">Title</span> {{task.Title}}
             </div>
             <div class="col-md-2 col-2 px-2">
-                <span class="material-icons">perm_identity</span> {{task.Assignee}}
+                <span class="material-icons" data-toggle="tooltip" style="vertical-align: middle;" data-placement="top" title="Assignee">perm_identity</span> {{task.Assignee}}
```
Comment (bighnesh0404) [inline worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.html:15]: add top padding

File: worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.html [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
+            <div class="col-md-1 col-1 px-2">
                 <app-status-icons [Status]="task.Status"></app-status-icons>
-            </div>
-            <div class="col-md-1 col-2 px-2">
                 <app-priority-icons [Priority]="task.Priority"></app-priority-icons>
             </div>
-            <div class="col-md-1 col-2 px-2">
-                {{task.id}}
+            <div class="col-md-1 col-1 px-2">
+                <span style="font-weight: bold;" data-toggle="tooltip" data-placement="top" title="task.id">{{task.id}}</span>
```
Comment (bighnesh0404) [inline worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.html:9]: add top padding

File: worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.html [hunk 3]
```diff
@@ -1,20 +1,18 @@
 <div class="card text-dark mb-3 taskCards" (click)="openTaskDetails(task.id)">
     <div class="card-header">
         <div class="row pb-2">
-            <div class="col-md-1 col-2 px-2">
+            <div class="col-md-1 col-1 px-2">
```
Comment (bighnesh0404) [inline worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.html:4]: add the top padding

**Review by bighnesh0404 — CHANGES_REQUESTED** [review]
Add top padding

File: worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.css [hunk 4]
```diff
@@ -1,5 +1,13 @@
 .taskCards {
+<<<<<<< HEAD
```
Comment (bighnesh0404) [inline worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.css:2]: remove the merge conflict code

File: worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.css [hunk 5]
```diff
@@ -1,5 +1,13 @@
...
+    border: 1px solid #8120f0;
+    font-size: 15px;
+=======
```
Comment (bighnesh0404) [inline worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.css:5]: remove it

File: worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.css [hunk 6]
```diff
@@ -1,5 +1,13 @@
...
     border: 1px solid #5A20F0;
     font-size: 12px;
+>>>>>>> fe1c7bac036e9b6f1042e4a14b8dfc03e2d86c21
```
Comment (bighnesh0404) [inline worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.css:8]: remove this

**Review by bighnesh0404 — CHANGES_REQUESTED** [review]
merge conflict needs to be done properly otherwise it will create problem later

**Review by bighnesh0404 — APPROVED** [review]
Ship it

File: worktrolly-ui/src/app/icons/priority-icons/priority-icons.component.html [hunk 7]
```diff
@@ -1,17 +1,17 @@
 <ng-container *ngIf="Priority == 'Low' ">
-    <span class="material-icons">
+    <span class="material-icons" style="color: yellow; padding: right 16px;" data-toggle="tooltip" data-placement="top" title="Low">
```
Comment (viv9k) [inline worktrolly-ui/src/app/icons/priority-icons/priority-icons.component.html:2]: Move css to .css file, use color code for color, and fix padding right. It's not a right way to use padding.

File: worktrolly-ui/src/app/icons/priority-icons/priority-icons.component.html [hunk 8]
```diff
@@ -1,17 +1,17 @@
...
         keyboard_arrow_down
     </span>
 </ng-container>
 
 <ng-container *ngIf="Priority == 'Medium' ">
-    <span class="material-icons">
+    <span class="material-icons" style="color: violet; padding: right 16px;" data-toggle="tooltip" data-placement="top" title="Medium">
```
Comment (viv9k) [inline worktrolly-ui/src/app/icons/priority-icons/priority-icons.component.html:8]: Move css to .css file, use color code for color, and fix padding right. It's not a right way to use padding.

File: worktrolly-ui/src/app/icons/priority-icons/priority-icons.component.html [hunk 9]
```diff
@@ -1,17 +1,17 @@
...
         sync_alt
     </span>
 </ng-container>
 
 <ng-container *ngIf="Priority == 'High' ">
-    <span class="material-icons">
+    <span class="material-icons" style="color: red; padding: right 16px;" data-toggle="tooltip" data-placement="top" title="High">
```
Comment (viv9k) [inline worktrolly-ui/src/app/icons/priority-icons/priority-icons.component.html:14]: Move css to .css file, use color code for color, and fix padding right. It's not a right way to use padding.

File: worktrolly-ui/src/app/icons/status-icons/status-icons.component.html [hunk 10]
```diff
@@ -1,29 +1,29 @@
 <ng-container *ngIf="Status == 'Completed' ">
-    <span class="material-icons">
-        check_circle_outline
+    <span class="material-icons" style="color: lime; padding-right:20px;" data-toggle="tooltip" data-placement="top" title="completed">
```
Comment (viv9k) [inline worktrolly-ui/src/app/icons/status-icons/status-icons.component.html:2]: Move css to .css file, use color code for color.

File: worktrolly-ui/src/app/icons/status-icons/status-icons.component.html [hunk 11]
```diff
@@ -1,29 +1,29 @@
...
+    check_circle_outline
     </span>
 </ng-container>
 
 <ng-container *ngIf="Status == 'Ready to start' ">
-    <span class="material-icons">
+    <span class="material-icons" style="color: rgb(255, 0, 128); padding-right:20px;" data-toggle="tooltip" data-placement="top" title="Ready to start">
```
Comment (viv9k) [inline worktrolly-ui/src/app/icons/status-icons/status-icons.component.html:8]: Move css to .css file, use color code for color.

File: worktrolly-ui/src/app/icons/status-icons/status-icons.component.html [hunk 12]
```diff
@@ -1,29 +1,29 @@
...
         not_started
     </span>
 </ng-container>
 
 <ng-container *ngIf="Status == 'Blocked' ">
-    <span class="material-icons">
+    <span class="material-icons" style="color: gray; padding-right:20px;" data-toggle="tooltip" data-placement="top" title="Blocked">
```
Comment (viv9k) [inline worktrolly-ui/src/app/icons/status-icons/status-icons.component.html:14]: Move css to .css file, use color code for color.

File: worktrolly-ui/src/app/icons/status-icons/status-icons.component.html [hunk 13]
```diff
@@ -1,29 +1,29 @@
...
         block
     </span>
 </ng-container>
 
 <ng-container *ngIf="Status == 'Under Progress' ">
-    <span class="material-icons">
+    <span class="material-icons" style="color: #fc6a03; padding-right:20px;" data-toggle="tooltip" data-placement="top" title="Under Progress">
```
Comment (viv9k) [inline worktrolly-ui/src/app/icons/status-icons/status-icons.component.html:20]: Move css to .css file, use color code for color.

File: worktrolly-ui/src/app/icons/status-icons/status-icons.component.html [hunk 14]
```diff
@@ -1,29 +1,29 @@
...
         arrow_circle_up
     </span>
 </ng-container>
 
 <ng-container *ngIf="Status == 'Ice Box' ">
-    <span class="material-icons">
+    <span class="material-icons" style="color: skyblue; padding-right:20px;" data-toggle="tooltip" data-placement="top" title="Ice Box">
```
Comment (viv9k) [inline worktrolly-ui/src/app/icons/status-icons/status-icons.component.html:26]: Move css to .css file, use color code for color.

File: worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.css [hunk 15]
```diff
@@ -1,5 +1,5 @@
-.taskCards {
-    border: 1px solid #5A20F0;
-    font-size: 12px;
-    color: #000;
-}
\ No newline at end of file
+.taskCards  {
+    border:1px solid #8120f0;
+    font-size:15px;
```
Comment (viv9k) [inline worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.css:3]: Let's not increase the font-size.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

File: worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.css (hunk 15 above)
Comment (shreyachowdhury4) [reply worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.css]: I have done,could you please review it

**Review by viv9k — APPROVED** [review]
ship it

**Pull request merged**